src/routes/Profile/get-referrer-link.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+
+export const getReferrerLink = async (account: string) => {
+  const baseUrl = process.env.REACT_APP_REFERRAL_AND_SIGNATURE_SERVICE_URL;
```

Comment:
As discussed I am defaulting this in https://github.com/Voltz-Protocol/voltz-ui/pull/375 if not set, cos that seemed better than errors or weird behaviour that we might not catch in testing. Totally open to not defaulting but we should probably be consistent.

Reply:
Me and @0xZenus created ticket on this matter. We will have a look into this in separate ticket addressing most of ENVs